dace/codegen/targets/rtl.py

```diff
@@ -0,0 +1,519 @@
+                                                     edge.dst.data)
+            elif isinstance(src_node.out_connectors[edge.src_conn], dace.vector):  # vector accessor
+                line: str = "{} {} = *({} *)(&{}[0]);".format(src_node.out_connectors[edge.src_conn].ctype,
+                                                              edge.src_conn,
+                                                              src_node.out_connectors[edge.src_conn].ctype,
+                                                              edge.dst.data)
+            else:  # scalar accessor
+                line: str = "{}* {} = &{}[0];".format(src_node.out_connectors[edge.src_conn].ctype, edge.src_conn,
+                                                      edge.dst.data)
+        else:
+            raise RuntimeError(
+                "Not handling define_out_memlet case of type {} -> {}.".format(type(edge.src), type(edge.dst)))
+        # write accessor to file
+        callsite_stream.write(line)
+
+    def get_generated_codeobjects(self):
+        """
+            Return list of code objects (that are later generating code files).
+        """
+        return self.code_objects
+
+    @property
+    def has_initializer(self):
+        """
+            Disable initializer method generation.
+        """
+        return False
+
+    @property
+    def has_finalizer(self):
+        """
+            Disable exit/finalizer method generation.
+        """
+        return False
+
+    @staticmethod
+    def cmake_options():
+        """
+            Process variables to be exposed to the CMakeList.txt script.
+        """
+        # get flags from config
+        verbose = Config.get_bool("compiler", "rtl", "verbose")
+        verilator_flags = Config.get("compiler", "rtl", "verilator_flags")
+        verilator_lint_warnings = Config.get_bool("compiler", "rtl", "verilator_lint_warnings")
+        # create options list
+        options = [
+            "-DDACE_RTL_VERBOSE=\"{}\"".format(verbose),
+            "-DDACE_RTL_VERILATOR_FLAGS=\"{}\"".format(verilator_flags),
+            "-DDACE_RTL_VERILATOR_LINT_WARNINGS=\"{}\"".format(verilator_lint_warnings)
+        ]
+        return options
+
+    def generate_rtl_parameters(self, constants):
+        # construct parameters module header
+        if len(constants) == 0:
+            return str()
+        else:
+            return "#(\n{}\n)".format(" " + "\n".join(
+                ["{} parameter {} = {}".format("," if i > 0 else "", key, constants[key]) for i, key in
+                 enumerate(constants)]))
+
+    def check_issymbolic(self, iterator: iter, sdfg):
+        for item in iterator:
+            # catch symbolic (compile time variables)
+            if symbolic.issymbolic(item, sdfg.constants):
+                raise RuntimeError("Please use sdfg.specialize to specialize the symbol in expression: {}".format(item))
```

Comment:
These functions doesn't use `self`, so they should not be in the class. Move them out as free functions